Command/EventLogCleanupCommand.php

```diff
@@ -13,7 +13,7 @@
 
 class EventLogCleanupCommand extends Command
 {
-    protected static $defaultName = 'mautic:leuchtfeuer:housekeeping';
+    protected static $defaultName = 'Leuchtfeuer:housekeeping';
```

Comment:
IMO the command name should be lower-case

Reply:
Changed to lower case with commit 3979ef0